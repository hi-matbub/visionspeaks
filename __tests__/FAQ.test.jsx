import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import FAQ from '../lib/static/FAQ.mdx';

it('FAQComponent components renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<FAQ/>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('FAQComponent component renders unchanged', () => {
  const tree = renderer.create(<FAQ/>).toJSON();
  expect(tree).toMatchSnapshot();
});
