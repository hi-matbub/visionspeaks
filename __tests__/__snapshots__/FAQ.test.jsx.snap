// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQComponent component renders unchanged 1`] = `
<h2>
  FAQ
</h2>
`;
